src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -273,6 +285,10 @@ def ComputeMoveLists(self):
       # Combine the individual point move lists
       move_list = set().union(*move_list)
       nbor_list = set().union(*nbor_list)
+      include_grants = ml.filterGrantsForFreqRange(
```

Comment:
Serge,
    Is this being done for all the Channels or just the OOB channels?

Reply:
This is done for all the channels. Otherwise you may end up with CBSD inside the DPA which are not in the move list.
This filterGrantsForFreqRange() function returns only the grant overlapping with the protection channel when inband (3550 < dpa_chan < 3650) and all of them when OOB (dpa_chan < 3550).

Reply:
If we are not pre-filtering the Grants, then for the grants overlapping with the protection channel, whether inside or outside, will already be on the ML as they do not have a 0km neighborhood distance.

Reply:
I am sorry I do not fully follow you.
A grant inside the DPA can have a neighborhood distance of 0km (like the CatA OOB currently) and would not be detected as part of the move list in the prior implementation, because a move list is the aggregation of all the indivcidual point move list. 
This PR fixes this by adding all those grants to the move list.

Similarly a grant with a neighborhood bigger than 0km (for example all others, Cat A and CatB inband, CatB OOB) could have a big pathloss to the nearby protection points (think about China Lake with mountains where the CBSD is in a valley and the protected point outside the valley), and NOT BE in the move list as a consequence.
This PR fixes this by making sure that they are all in the move list, like if there was a protected point right under the CBSD.

Let me know if I misunderstood your comment.

Reply:
Serge,
   That is a good point. I agree with you that this would cover all the cases and not leave anything to be dependent on the interference to be large when CBSDs are within the DPA and hence will automatically be put on the ML.
I agree and approve the changes